app/signals/apps/search/tasks.py

```diff
@@ -86,3 +89,30 @@ def index_signals_updated_in_date_range(from_date=None, to_date=None):
     SignalDocument.index_documents(queryset=signal_qs)
 
     log.info('index_signals_updated_in_date_range - done!')
+
+
+@app.task
+def index_status_message(status_message_id: int):
+    """Celery task that indexes a status message.
+
+    Parameters
+    ----------
+    status_message_id : int
+        The database id of the status message to be indexed.
+    """
+    status_message = StatusMessageModel.objects.get(id=status_message_id)
```

Comment:
Maybe add a try except here, it should not happen but it could be possible this task will be executed and the StatusMessage is already deleted.

Reply:
If it's deleted the task will fail right? Would that crash the worker or something? Otherwise it not being indexed is exactly what should happen I guess.

Reply:
As discussed, added exception handling.